src/app/screens/Accounts/index.tsx

```diff
@@ -195,6 +236,70 @@ function AccountsScreen() {
             </div>
           </form>
         </Modal>
+
```

Comment:
@dylancom you think by now we should move out these two modals into separate component-files?